package.json

```
@@ -2,48 +2,48 @@
   "name": "laboratorium-ee-2019",
   "version": "0.0.1",
   "private": true,
+  "engines": {
+    "node": "16.x"
+  },
```

Chyba mieliśmy się na razie trzymać LTS :p
```suggestion
  "engines": {
    "node": "14.x"
  },
```

---

od niedawna lts to 16

---

O kurde, to trzeba przenieść do generatorów